docs/_includes/toc.md

```diff
@@ -84,24 +76,24 @@
       </ul>
     </details>
     <details>
-      <summary>API Documentation</summary>
+      <summary>Design and Philosophy</summary>
       <ul>
-        <li><a href="/fprime/UsersGuide/dev/gds-cli-dev.html">GDS CLI Design</a></li>
-        <li><a href="/fprime/UsersGuide/api/c++/html/index.html">C++ Documentation</a></li>
-        <li><a href="/fprime/UsersGuide/cmake/cmake-api.html">CMake User API</a></li>
+        <li><a href="/fprime/Design/fprime-architecture.html">F´ Software Architecture</a></li>
+        <li><a href="/fprime/Design/numerical-types.html">Numerical Types Design</a></li>
+        <li><a href="/fprime/Design/communication-adapter-interface.html">Communication Adapter Interface</a></li>          <li><a href="/fprime/Design/package-implementations.html">Package Implementations</a></li>
+        <li><a href="/fprime/Design/fpp-json-dict.html">FPP JSON Dictionary Specification</a></li>
       </ul>
     </details>
   </ul>
-<h4><a href="/fprime/Design/general.html">Design and Philosophy</a></h4>
-  <ul>
-    <li><a href="/fprime/Design/fprime-architecture.html">F´ Software Architecture</a></li>
-    <li><a href="/fprime/Design/numerical-types.html">Numerical Types Design</a></li>
-    <li><a href="/fprime/Design/communication-adapter-interface.html">Communication Adapter Interface</a></li>
-    <li><a href="/fprime/Design/package-implementations.html">Package Implementations</a></li>
-    <li><a href="/fprime/Design/fpp-json-dict.html">FPP JSON Dictionary Specification</a></li>
-  </ul>
-<h4><a href="/fprime/Design/general.html">How-To Guides</a></h4>
+<h4><a href="/fprime/HowTo/general.html">How-To Guides</a></h4>
   <ul>
     <li><a href="/fprime/HowTo/develop-gds-plugins.html">F´ GDS Plugin Development</a></li>
     <li><a href="/fprime/HowTo/develop-fprime-libraries.html">F´ Library Development</a></li>
   </ul>
+<h4><a href="/fprime/Reference/README.html">Reference</a>      
```

Comment:
`<h4>` should be closed at the end of the line instead of encapsulating the whole block

Reply:
Oops. This one should be fixed now.